### PR TITLE
Create PR_16

### PR DESCRIPTION
**Pull request by OksanaEfanova**
desc

### CODE REVIEW

File: PR_16 [hunk 1]
```diff
@@ -0,0 +1 @@
+PR_16
```
Comment (TestGitHub100500) [inline PR_16:1]: somecomment

Comment (TestGitHub100500) [inline PR_16:1]: somecomment2

Comment (OksanaEfanova) [reply PR_16:1]: response

Comment (OksanaEfanova) [reply PR_16:1]: response2

**Comment by TestGitHub100500** [comment]
comment1

File: PR_16 (hunk 1 above)
Comment (TestGitHub100500) [reply PR_16:1]: somecomment3

**Comment by OksanaEfanova** [comment]
A+

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]
RCH

**Comment by OksanaEfanova** [comment]
comment

**Review by TestGitHub100500 — CHANGES_REQUESTED** [review]
RCH2

**Comment by OksanaEfanova** [comment]
comment

**Pull request closed without merging**